source/slang/slang-lower-to-ir.cpp

```diff
@@ -1440,14 +1440,14 @@ static void addLinkageDecoration(
     const String mangledName = getMangledName(context->astBuilder, decl);
 
     // Obfuscate the mangled names if necessary.
-    // 
-    // Care is needed around stdlib as it is only compiled once and *without* obfuscation, 
+    //
+    // Care is needed around stdlib as it is only compiled once and *without* obfuscation,
     // so any linkage name to stdlib *shouldn't* have obfuscation applied to it.
-    if (context->shared->m_obfuscateCode && 
```

Comment:
Even though this is a random space at the end of a line, we should not be removing random white space.

Reply:
Noted. Some the files right now (lower-to-ir included) are modified just for debugging. I will revert them to the original state when I finish implementing and testing.